test/paths.js

```diff
@@ -5,75 +5,75 @@ var meteorResolver = require('../index.js')
 
 describe('paths', function () {
   it('handles base path relative to CWD', function () {
-    expect(meteorResolver.resolve('./another-file', './test/imports/test-file.js'))
+    expect(meteorResolver.resolve('./another-file', './test/imports/test-file.js'.replace(/\//g, path.sep)))
       .to.have.property('path')
       .equal(path.resolve(__dirname, './imports/another-file.js'))
   })
 
   it('handles root (/) paths relative to CWD', function () {
-    expect(meteorResolver.resolve('/imports/another-file', './test/imports/test-file.js'))
+    expect(meteorResolver.resolve('/imports/another-file', './test/imports/test-file.js'.replace(/\//g, path.sep)))
       .to.have.property('path')
       .equal(path.resolve(__dirname, './imports/another-file.js'))
   })
 
   it('should not resolve a client file in a server file', function () {
-    expect(meteorResolver.resolve('/imports/client/client-test', './test/imports/server/server-test.js'))
+    expect(meteorResolver.resolve('/imports/client/client-test', './test/imports/server/server-test.js'.replace(/\//g, path.sep)))
       .to.deep.equal({found: false})
   })
 
   it('should not resolve a client file in a non-client file', function () {
-    expect(meteorResolver.resolve('/imports/client/client-test', './test/imports/package-test/plain-file.js'))
+    expect(meteorResolver.resolve('/imports/client/client-test', './test/imports/package-test/plain-file.js'.replace(/\//g, path.sep)))
```

Comment:
Can you turn this into a function? You copied the same code on every single test. Makes sense to abstract it.